package.json

```diff
@@ -44,13 +44,13 @@
 		"@emotion/babel-preset-css-prop": "^11.12.0",
 		"@guardian/eslint-config-typescript": "^12.0.0",
 		"@guardian/prettier": "^8.0.1",
-		"@storybook/addon-essentials": "^8.2.9",
-		"@storybook/addon-links": "^8.2.9",
+		"@storybook/addon-essentials": "^8.3.5",
+		"@storybook/addon-links": "^8.3.5",
 		"@storybook/addon-styling-webpack": "^1.0.0",
-		"@storybook/addon-themes": "^8.2.9",
+		"@storybook/addon-themes": "^8.3.5",
 		"@storybook/addon-webpack5-compiler-babel": "^3.0.3",
-		"@storybook/react": "^8.2.9",
-		"@storybook/react-webpack5": "^8.2.9",
+		"@storybook/preact-webpack5": "^8.3.5",
+		"@storybook/react": "^8.3.5",
```

Comment:
Do we possibly want to use the [@storybook/preact](https://www.npmjs.com/package/@storybook/preact) package here instead aswell?

Reply:
Oh nevermind 😂 

![Screenshot 2024-10-16 at 16 28 40](https://github.com/user-attachments/assets/44493172-5f1e-4759-8a1b-43b69cddb36f)